tests/lib/rules/comma-dangle.js

```diff
@@ -392,8 +392,8 @@ ruleTester.run("comma-dangle", rule, {
                 {
                     message: "Missing trailing comma.",
                     type: "Property",
-                    line: 2,
-                    column: 11
+                    line: 3,
+                    column: 1
```

Comment:
Ah, good point about `comma-style`. Agree with you, though, that `comma-style: last` is the more commonly used config.